python_modules/dagster-graphql/dagster_graphql_tests/graphql/test_asset_checks.py

```diff
@@ -759,3 +759,43 @@ def test_launch_subset_asset_and_included_check(self, graphql_context: Workspace
             and log.dagster_event.event_type == DagsterEventType.ASSET_MATERIALIZATION
         ]
         assert len(materializations) == 1
+
+    def test_launch_subset_asset_no_check(self, graphql_context: WorkspaceRequestContext):
+        selector = infer_job_or_pipeline_selector(
+            graphql_context,
+            "asset_check_job",
+            asset_selection=[{"path": ["asset_1"]}],
+            asset_check_selection=[],
```

Comment:
existing `None` coverage is sufficient?

Reply:
yeah we test it in test_launch_subset_asset_and_included_check